UnityProject/Assets/Scripts/Items/Weapons/WeaponNetworkActions.cs

```diff
@@ -135,6 +145,8 @@ public void ServerPerformMeleeAttack(GameObject victim, Vector2 attackDirection,
 				if (victim.TryGetComponent<LivingHealthMasterBase>(out var victimHealth))
 				{
 					victimHealth.ApplyDamageToBodyPart(gameObject, damage, AttackType.Melee, damageType, damageZone);
+					victimHealth.ApplySlashDamage(slashChance, damageZone, slashDamage);
+					victimHealth.ApplyPierceDamage(pierceChance, damageZone, pierceDamage);
```

Comment:
surely this would be better sorted by attack type?

Reply:
tho I suppose you couldn't get separate damage values for that

Reply:
No, because slash and pierce are not damage types, they're things that can get applied by any attack or damage type.